packages/mobile/src/screens/signon/SignOn.tsx

```diff
@@ -665,6 +666,9 @@ const SignOn = ({ navigation }: SignOnProps) => {
   }
 
   const validateEmail = (email: string) => {
+    dispatch(signOnActionsWeb.setValueField('email', email))
```

Comment:
as far as pattern, do we want this to just be `signOnActions` and the other one be `signOnLegacyActions`, to follow pattern in the store?

Reply:
I called it signOnDeprecated, but that sounds better, ill switch everything to signOnLegacy